code/game/machinery/doors/airlock.dm

```diff
@@ -980,6 +1020,32 @@
 /obj/machinery/door/airlock/proc/weld_checks(obj/item/weldingtool/W, mob/user)
 	return !operating && density
 
+/**
+  * Used when attempting to remove a seal from an airlock
+  *
+  * Called when attacking an airlock with an empty hand, returns TRUE (there was a seal and we removed it, or failed to remove it)
+  * or FALSE (there was no seal)
+  * Arguments:
+  * * user - Whoever is attempting to remove the seal
+  */
+/obj/machinery/door/airlock/try_remove_seal(mob/living/user)
+	if(seal)
+		var/obj/item/door_seal/D = seal
```

Comment:
Better variable name

Reply:
done (if airlock seal counts as better, which to be fair, probably not)